mailpoet/tests/performance/tests/subscribers-filtering.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-shadow */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-default-export */
+/**
+ * External dependencies
+ */
+import { sleep, check, group } from 'k6';
+import { chromium } from 'k6/x/browser';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
```

Comment:
It might be nice to include helpers like this directly in our codebase, maybe in `helpers.js`?, just so we don't have to rely on fetching dependencies over the internet.

Reply:
This is by default and according to documentation to use it this way if working with k6.
Maybe only for the 3rd one, random number, but since i use it already in all tests one separate PR would be necessary for this change. I would leave it as-is in this PR.

Reply:
Sounds good! Definitely not a big deal.